src/main/java/ch/heigvd/dai/commands/Root.java

```diff
@@ -9,18 +9,28 @@
     scope = CommandLine.ScopeType.INHERIT,
     mixinStandardHelpOptions = true)
 public class Root {
+
   @CommandLine.Parameters(index = "0", description = "The path to the input file.")
   protected String inputPath;
 
-  // TODO : make optional ? If omitted, original picture would be replaced ?
-  @CommandLine.Parameters(index = "1", description = "The desired path for the output file.")
+  // Make outputPath optional with arity = "0..1"
+  @CommandLine.Parameters(
+      index = "1",
+      description = "The desired path for the output file.",
```

Comment:
Maybe add something saying a default one will be made if none is input ?

Reply:
I think it is not mandatory because the program shows only one required parameter when we try to execute it without any parameter. Like that 
Missing required parameter: '<inputPath>'
Usage: pictures-cli-editor-1.0-SNAPSHOT.jar [-hV] <inputPath> [<outputPath>]
       [COMMAND]
A CLI tool to edit BMP pictures.
      <inputPath>      The path to the input file.
      [<outputPath>]   The desired path for the output file.
  -h, --help           Show this help message and exit.
  -V, --version        Print version information and exit.
Commands:
  grey  Convert a picture to black and white
  blur  Blur a picture or part of it
  crop  Crop a picture to a new format

Reply:
Indeed